wafw00f/plugins/radware.py

```diff
@@ -15,4 +15,7 @@ def is_waf(self):
             b'<TITLE>Unauthorized Request Blocked</TITLE>', b'If you believe that there has been some mistake',
             b'?Subject=Security Page - Case Number')):
             return True
+        # Restored a fingerprint for radware previously discarded
+        if b'CloudWebSec@radware.com' in responsebody:
```

Comment:
This new fingerprint is more reliable. It was earlier discarded due to false negatives.